### DISCUSSION TITLE
Bypass Welcome page

### DISCUSSION THREAD

**Discussion by upintheairsheep** [yume-chan/ya-webadb#579] [Ideas]
Make a way to bypass the first prompt for users wanting to test out without owning an Android device, or if the device is not available.

**Comment by yume-chan** [comment]
How should it work if no device is connected?

**Comment by upintheairsheep** [comment]
A demo interface allowing the user to see the different options available

**Reply by CorruptionVR** [reply]
I think that this is a good idea. it would let people see what features it has before they connect their phone. Maybe add a "dummy device" that has preset values, such as fake files, folders and apps with preview images to simulate screen mirroring.
